astropy/wcs/wcs.py

```diff
@@ -388,8 +396,8 @@ def __init__(self, header=None, fobj=None, key=' ', minerr=0.0,
                         "header must be a string, an astropy.io.fits.Header "
                         "object, or a dict-like object")
 
-            header_string = header_string.strip()
-
```

Comment:
@mdboom Is there a reason this line was removed? If empty lines in the header don't get removed, calling `to_header` creates fits cards like `keyword=" " , value = None`. This came up just now because wcslib 5.8 adds an empty line and a comment line. Here's what I mean:
```
w.to_header()
Out[13]: 
WCSAXES =                    2 / Number of coordinate axes                      
CRPIX1  =               2048.0 / Pixel coordinate of reference point            
CRPIX2  =               1024.0 / Pixel coordinate of reference point            
PC1_1   =    1.29055156974E-05 / Coordinate transformation matrix element       
PC1_2   =    5.95250078647E-06 / Coordinate transformation matrix element       
PC2_1   =    5.02263819607E-06 / Coordinate transformation matrix element       
PC2_2   =   -1.26448441522E-05 / Coordinate transformation matrix element       
CDELT1  =                  1.0 / [deg] Coordinate increment at reference point  
CDELT2  =                  1.0 / [deg] Coordinate increment at reference point  
CUNIT1  = 'deg'                / Units of coordinate increment and value        
CUNIT2  = 'deg'                / Units of coordinate increment and value        
CTYPE1  = 'RA---TAN-SIP'       / TAN (gnomonic) projection + SIP distortions    
CTYPE2  = 'DEC--TAN-SIP'       / TAN (gnomonic) projection + SIP distortions    
CRVAL1  =        5.63056810618 / [deg] Coordinate value at reference point      
CRVAL2  =       -72.0545718428 / [deg] Coordinate value at reference point      
LONPOLE =                180.0 / [deg] Native longitude of celestial pole       
LATPOLE =       -72.0545718428 / [deg] Native latitude of celestial pole        
WCSNAME = 'IDC_postsm4'        / Coordinate system title                        
RADESYS = 'FK5'                / Equatorial coordinate system                   
EQUINOX =               2000.0 / [yr] Equinox of equatorial coordinates         
                                                                                
COMMENT  WCS header keyrecords produced by WCSLIB 5.8         
```
I have code that loops over WCS keywords and changes them. I'll change my code to look for empty lines and comments but just wondering if the previous behaviour is not preferable,